File: CC/Codigos_Fonte_Relatorio_1/micro03/micro03.pas
program micro03;

var
    numero : integer;

begin

    write('Digite um número');
    read(numero);

    if( numero >= 100) then
        if (numero <= 200) then 
            writeln('O número está no intervalo entre 100 e 200')

        else writeln('O número não está no intervalo entre 100 e 200')

    else writeln('O número não está no intervalo entre 100 e 200')

end.

File: CC/Codigos_Fonte_Relatorio_1/micro03/micro03.java

public class micro03{

    public static void main(String[] args){
        int numero;
        System.out.println("Digite um número: ");
        numero = Integer.parseInt(System.console().readLine());
    
        if(numero >= 100)
            if(numero <= 200) System.out.println("O número está no intervalo entre 100 e 200"); 
            else System.out.println("O número não está no intervalo entre 100 e 200"); 

        else System.out.println("O número não está no intervalo entre 100 e 200"); 

        
    }
}
